manifests/init.pp

```diff
@@ -144,6 +146,7 @@
   Variant[Stdlib::Absolutepath, Array[Stdlib::Absolutepath]] $pulpcore_additional_import_paths = [],
   Variant[Stdlib::Absolutepath, Array[Stdlib::Absolutepath]] $pulpcore_additional_export_paths = [],
   Boolean $pulpcore_telemetry = false,
+  Boolean $pulpcore_hide_guarded_distributions = false,
```

Comment:
```suggestion
  Optional[Boolean] $pulpcore_hide_guarded_distributions = undef,
```

I'm torn :)

Reply:
or `= true`?

Reply:
Both, Jameer and I think it should be `true` in the opinionated Katello deployment.

Reply:
I'm also leaning to `true`.

Reply:
it has been `true`d